src/coreclr/src/inc/cordebug.idl

```diff
@@ -2567,7 +2567,7 @@ typedef enum CorGCReferenceType
     CorHandleStrongDependent = 1<<6,
     CorHandleStrongAsyncPinned = 1<<7,
     CorHandleStrongSizedByref = 1<<8,
-    CorHandleWeakWinRT = 1<<9,
+    CorHandleWeakNativeCom = 1<<9,
```

Comment:
>CorHandleWeakNativeCom [](start = 4, length = 22)

Careful with this. The Diagnostic team needs to be tag to approve this change. It is technically breaking if we publish this API.

Reply:
Good catch. I'll tag them so they can take a look. Worst case we can just change the name back.

cc: @dotnet/dotnet-diag

Reply:
The idls are the contract, once they're published we shouldn't change them. Just changing the name and not the value isn't the worst case (existing code would run fine) but people would have compiler errors when they went to rebuild.

I think it would be better to leave it as is and add a comment saying it refers to the native com handle since winrt handles don't exist anymore.

Reply:
Another approach is to have the new enum have the same value as the previous one.
`CorHandleWeakNativeCom = 1<<9` and `CorHandleWeakWinRT = CorHandleWeakNativeCom`.

Reply:
I've restored the old enum with the same value.